lib/buffer-search.js

```diff
@@ -224,6 +224,8 @@ class BufferSearch {
       }
 
       const newMarkers = this.createMarkers(scanStart, scanEnd);
+      if(newMarkers === false || !newMarkers.length) return; // Invalid/missing find pattern or no new markers
```

Comment:
We've fixed this as part of https://github.com/atom/find-and-replace/pull/860. Didn't mean to steal your 🌩  but we're on a spree of uncaught exception fixes after publishing 1.14.